libs/labelbox/src/labelbox/schema/api_key.py

```diff
@@ -50,7 +50,9 @@ def created_by(self) -> Optional["User"]:
         if not hasattr(self, "_created_by"):
             self._created_by = (
                 self.client._get_single(User, self.created_by_user_id)
-                if self.created_by_user_id
+                if self.created_by_user_id is not None
+                else self.client._get_single(User, self.user_id)
+                if self.user_id is not None
                 else None
```

Comment:
This statement is very confusing

Reply:
I've modified the logic to make it more understandable.